pyproject.toml

```diff
@@ -16,3 +16,152 @@ multi_line_output = 3
 include_trailing_comma = true
 use_parentheses = true
 skip = "migrations"
+
+[tool.poetry]
+name = "redash"
+version = "23.09.0"
```

Comment:
Does poetry not understand things like `-dev` in version identifiers?

Reply:
Resolved!